src/content_script.js

```diff
@@ -304,127 +406,74 @@ function addFacebookBadge (target, badgeClassUId, socialAction) {
 
   const ratioCheck = (itemWidth / itemHeight);
 
-  let allowClickSwitch = false;
 
   const badgeSmallSwitch = shouldBadgeBeSmall(ratioCheck, itemHeight);
   if (badgeSmallSwitch) {
     htmlBadgeDiv.classList.add("fbc-badge-small");
   }
 
-  positionFacebookBadge(target, badgeClassUId, itemWidth, badgeSmallSwitch);
+  const FBC_IFRAME_HEIGHT_LOGIN = 230;
+  const FBC_IFRAME_HEIGHT_EMAIL = 240;
 
   // Show/hide prompt if login element
-  if (socialAction === "login"){
-    target.addEventListener("click", (e) => {
+  if (socialAction === "login") {
```

Comment:
suggestion (non-blocking): this might be better as a `switch` statement since we only handle the 4 distinct values for `socialAction`?

Reply:
Done.